src/index.tsx

```diff
@@ -216,7 +216,7 @@ class SvgRenderer extends React.PureComponent<Props, State> {
       );
     }
 
-    return null;
+    return <View />;
```

Comment:
Why did you add this change? This will actively add and render a component in the view stack, while returning `null` wouldn't render anything.

Reply:
Because when using the `SvgRenderer` along with animations (`Animated.createAnimatedComponent(SvgRenderer)`) the view is detached and causes a crash. This was the only way I could make it work properly. Do you have any other ideas of how to fix that without returning a `<View />`?